frontend/src/modules/entitydetails/selectors.js

```diff
@@ -5,12 +5,15 @@ import { createSelector } from 'reselect';
 import * as navigation from 'core/navigation';
 import * as plural from 'core/plural';
 import * as entitieslist from 'modules/entitieslist';
+import * as user from 'core/user';
 
+import type { DbEntity, Entities } from 'modules/entitieslist';
```

Comment:
Those should be sorted by module name, so in this case it should come after the `core/` module.

Reply:
Also, `DbEntity` is never used in this file.